TombEngine/Specific/Worker.cpp

```diff
@@ -0,0 +1,144 @@
+#include "framework.h"
+#include "Specific/Worker.h"
+
+namespace TEN::Utils
+{
+	WorkerManager g_Worker = WorkerManager();
+
+	WorkerManager::WorkerManager()
+	{
+		// Reserve threads.
+		unsigned int threadCount = std::thread::hardware_concurrency() * 2;
```

Comment:
Replace the call to std::thread::hardware_concurrency with a call to GetCoreCount(), considering you already use that to alias std::thread::hardware_concurrency